cli/shared/src/main/scala/com/lightbend/rp/reactivecli/docker/DockerRegistry.scala

```diff
@@ -21,23 +21,28 @@ import argonaut._
 import com.lightbend.rp.reactivecli.concurrent._
 import com.lightbend.rp.reactivecli.http._
 import com.lightbend.rp.reactivecli.http.Http.HttpExchange
+import com.lightbend.rp.reactivecli.Platform
 import scala.concurrent.{ ExecutionContext, Future }
 import scala.util.{ Failure, Success, Try }
 import scalaz._
 import slogging._
+import fastparse.all._
 
 import Argonaut._
 import Scalaz._
 
 object DockerRegistry extends LazyLogging {
   private[docker] def blobUrl(img: Image, digest: String, useHttps: Boolean): String =
-    s"${protocol(useHttps)}://${img.url}/v2/${img.namespace}/${img.image}/blobs/$digest"
+    Platform.encodeURI(s"${protocol(useHttps)}://${img.url}/v2/${img.namespace}/${img.image}/blobs/$digest")
```

Comment:
Further improvement would be to use a URI builder, like e.g. in Akka HTTP it would be something like `Uri(img.url) / "v2" / img.namespace / img.image / "blobs" / digest`, allowing each path component to be properly encoded.

We don't have such an API yet, but just writing this to note.